snakemake/__init__.py

```diff
@@ -2486,6 +2493,25 @@ def get_argument_parser(profile=None):
         "If you don't have a shared filesystem, additionally specify --no-shared-fs.",
     )
 
+    group_flux_operator.add_argument(
+        "--flux-operator",
+        action="store_true",
+        help="Execute your workflow on a flux MiniCluster cluster using the Flux Operator.",
+    )
+
+    group_flux_operator.add_argument(
+        "--flux-operator-namespace",
+        default="flux-operator",
+        help="Namespace for the Flux Operator MiniCluster(s).",
+    )
+    group_flux_operator.add_argument(
+        "--container-image",
+        metavar="IMAGE",
+        help="Docker image to use, e.g., when submitting jobs to the Flux MiniCluster"
+        "on Kubernetes. It must contain a Flux install and have active user root. "
+        "Defaults to ghcr.io/rse-ops/mamba:app-mamba",
+    )
```

Comment:
There is already the --container-image argument, which can be also used for the flux operator.

Reply:
I tried using the same name, but it got angry at me. Are you saying to remove it from the kubernetes group and add to the flux operator group? or is it possible to define the same param for two groups?